### ISSUE TITLE
Disk decryption password prompt disappears after applying the suggested update on a fresh install

### ISSUE THREAD

**Issue by segmentationsingularity** [pop-os/plymouth-theme#31]
Hi There,

Just wondering if anybody else is having the same issue as I am having, 
I've installed POP OS version 22.04 (Nvidia version) with disk encryption enabled on a new system (AMD dual CPU, 3090Ti) single Nvme SSD and everything works fine (uefi mode), also on reboots etc.
However when I accept and install the suggested updates (from the Pop OS shop, clicked "Update all") and restart the system as suggested by the Pop OS Shop. 
However, the decryption prompt doesn't show up, only the GIGABYTE logo, I've tried pressing esc (2x) and that doesn't do anything, however when I enter the disk decryption password the system does boots and everything works fine. 
Note that I haven't modified or installed anything (Only the suggested updates from the Pop OS Shop)
In short:
1. Plain vanilla install -> everything works like a charm, (Gigabyte logo appears and shortly after the decryption screen shows up)
2. Accept the updates -> no decryption prompt (The Gigabyte logo is the only thing shown)

I've tried a couple of things like described on: 
1) https://support.system76.com/articles/login-loop-pop/#recovery
2) checked if my crypttab file (as described here: https://github.com/pop-os/plymouth-theme/issues/19#issuecomment-875149392) for unexpected entries (like the nofail flag) but the file was the same (before and after installing the updates) 

Suggestions would be most welcome.

**Comment by leviport** [comment]
We've been seeing this occasionally happen on Nvidia desktops with full disk encryption. It does appear that entering your encryption password after waiting a bit still unlocks the disk as intended. The other option is to boot the previous kernel. To select the previous kernel, hold spacebar while the machine is booting up, until the systemd-boot menu appears. You should see a current kernel option and an oldkern option. Selecting one of these options is a one-time thing, but you can set one as the default by moving the selector to it and pressing the `d` key. Just remember to un-set this override once the fix is available so you're not perpetually behind by a kernel version.

**Comment by segmentationsingularity** [comment]
That did the trick! the only difference visually is that the encryption screen seems to be displayed in a lower resolution, however happy to use this workaround!
Thanks!

**Comment by leviport** [comment]
Glad that helped! However, I'd like to leave this issue open while we work on a proper fix for the bug.

**Comment by andradewall** [comment]
Same here. 

> We've been seeing this occasionally happen on Nvidia desktops with full disk encryption. It does appear that entering your encryption password after waiting a bit still unlocks the disk as intended. The other option is to boot the previous kernel. To select the previous kernel, hold spacebar while the machine is booting up, until the systemd-boot menu appears. You should see a current kernel option and an oldkern option. Selecting one of these options is a one-time thing, but you can set one as the default by moving the selector to it and pressing the `d` key. Just remember to un-set this override once the fix is available so you're not perpetually behind by a kernel version.

Thank you for sharing this _solution_, but unfortunately this doesnt fix the bug...

**Comment by leviport** [comment]
@andradewall I mentioned that we are still working on a proper fix...

**Comment by leviport** [comment]
Update: I believe https://github.com/pop-os/linux/pull/177 fixed this for most machines, but some are still affected. We are continuing to pursue a fix for all machines.

**Comment by leviport** [comment]
https://github.com/pop-os/linux/pull/178 should fix this for all machines. It has been released, so regular updates should bring you the fix.

**Comment by leviport** [comment]
@segmentationsingularity can you please describe the hardware you're seeing this on? It seems to have gone away on the machines I've been testing on, so getting a setup more similar to yours would be helpful for debugging.

**Comment by segmentationsingularity** [comment]
Thanks for reopening this :+1:  this is on a gigabyte mz72-hb0 rev 3/4 with 2 AMD EPYC 7v13 (milan) 64Core CPU's
and a MSI GeForce RTX 3090 SUPRIM X 24G

**Comment by segmentationsingularity** [comment]
After installing kernel 6.1.11 (pop-os) the decryption prompt reappeared!
Note: I didn't change the Nvidia driver (pop-os) 525.85.05
